src/Sylius/Bundle/PricingBundle/Form/EventListener/BuildPriceableFormSubscriber.php

```diff
@@ -85,8 +86,13 @@ public function preBind(FormEvent $event)
      * @param string        $calculatorType
      * @param array         $data
      */
-    protected function addConfigurationFields(FormInterface $form, $calculatorType, array $data = array())
+    protected function addConfigurationFields(FormInterface $form, $calculatorType = null, array $data = array())
```

Comment:
Our situation is we're posting data to product's API route, right now we only want to support `Standard` calculator so our developers expect that if they don't pass a calculator it becomes `standard` by default.

The way that resource bundle is handling the the API's content is using Forms so it means that product form will be created with an empty (null) pricingCalculator, hence on [this line](https://github.com/Sylius/Sylius/pull/3061/files#diff-1676b06edd6dab16504a09f9c110dc84L78) `$data['pricingCalculator']` is null which is breaking things.

If you think it is OK I can check for null value on that line!?